### PR TITLE
imagemagick - bump pango dep AND remove xft hack

### PR DESCRIPTION
**Pull request by paulharris**
Pango now depends on the new libxft recipe, which removes the need for
this workaround/hack.

Part of the series to move libxft out of xorg/system

See PR #10900 

These should all be merged together.

### CODE REVIEW

**Comment by paulharris** [comment]
Any idea why the build is failing when it looks for the prebuilt binaries?
ERROR: Missing prebuilt package for 'imagemagick/7.0.11-14'

It is supposed to be building this!  Not installing the prebuilts...

**Comment by ericLemanissier** [comment]
LGTM, but this should be merged after https://github.com/conan-io/conan-center-index/pull/10902

File: recipes/imagemagick/all/conanfile.py [hunk 1]
```diff
@@ -327,17 +327,6 @@ def _build_configure(self):
             self, win_bash=tools.os_info.is_windows
         )
 
-        # FIXME: workaround for xorg/system adding system includes https://github.com/conan-io/conan-center-index/issues/6880
-        xft_path = os.path.join(self.build_folder, "xft.pc")
-        if os.path.exists(xft_path):
-            os.remove(xft_path)
-        if "/usr/include/uuid" in self._autotools.include_paths:
-            self._autotools.include_paths.remove("/usr/include/uuid")
```
Comment (ericLemanissier) [inline recipes/imagemagick/all/conanfile.py:335]: unfortunately I think these two lines cannot be removed yet, as `uuid` is part of xorg/system package (needed for `sm` component)

Comment (paulharris) [reply recipes/imagemagick/all/conanfile.py]: ok, re-added

**Review by ericLemanissier — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Pull request merged**